test/passing/tests/object_expr.ml.ref

```diff
@@ -0,0 +1,16 @@
+(object
+   method one = 1
+end )
```

Comment:
Unrelated, there's a bug in these parentheses: there should either be no spacing or a space both at the opening and closing.

Reply:
Noted, but I think there is already an issue open about that. It might cause a big diff though, we can take care of this for the next release.